extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -35,23 +37,26 @@ export class NetCoreTool {
 	private static _outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel(projectsOutputChannel);
 
 	public async findOrInstallNetCore(): Promise<boolean> {
-		if (!this.isNetCoreInstallationPresent) {
+		if (!this.isNetCoreInstallationPresent && vscode.workspace.getConfiguration(DBProjectConfigurationKey)[NetCoreDoNotAskAgainKey] !== true) {
 			await this.showInstallDialog();
 			return false;
 		}
 		return true;
 	}
 
 	public async showInstallDialog(): Promise<void> {
-		let result = await vscode.window.showInformationMessage(NetCoreInstallationConfirmation, UpdateNetCoreLocation, InstallNetCore);
+		let result = await vscode.window.showInformationMessage(NetCoreInstallationConfirmation, UpdateNetCoreLocation, InstallNetCore, DoNotAskAgain);
```

Comment:
Something about this ordering feels a bit weird now that "Don't Ask" is included... @radentonev, is there a preferred order for change/positive/ignore buttons?

Reply:
This article was an neat read: https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150

Reply:
I like Karl's idea too. For real customers, not tests or internal folks testing, I do think the install options are more important to show up front so their builds dont fail...

Reply:
Looks like that option isn't exposed to extensions; primary actions only.